### PR TITLE
Create GetWaterML method

### PR DESCRIPTION
**Pull request by msouff**
This creates a waterml file for a specific forecast statistic (mean, std dev, outer range, or high res). This method does not interfere with previous SPT functionality. It just adds a way to get results as waterml so we can use them as input for third party web apps.

### CODE REVIEW

**Comment by snowman2** [comment]
This looks great. Would you mind moving this pull request to the dev branch?

**Comment by msouff** [comment]
Done.

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ American Water Resources Association (JAWRA)* 1-15, DOI: 10.1111/1752-1688.12434
 
 Snow, Alan Dee, "A New Global Forecasting Model to Produce High-Resolution Stream Forecasts" (2015). All Theses and Dissertations. Paper 5272. http://scholarsarchive.byu.edu/etd/5272
 
+<<<<<<< HEAD
```
Comment (snowman2) [inline README.md:17]: These tags shouldn't be there. I think the conflict fix didn't work. Can you try again?

**Comment by snowman2** [comment]
Also, can you add a blurb in the documentation about how to query for data?

**Pull request closed without merging**